### PR TITLE
create option Privilegios INUBE

### PR DESCRIPTION
**Pull request by SelsaErojas**
create option Privilegios INUBE

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,24 +7,27 @@ import {
 
 import { ErrorPage } from "@components/layout/ErrorPage";
 import { AppPage } from "@components/layout/AppPage";
-
+//import AppContextProvider, { AppContext } from "@context/AppContext";
```
Comment (ANGIECATHERINEP) [inline src/App.tsx:10]: you should remove the comments

File: src/App.tsx [hunk 2]
```diff
@@ -7,24 +7,27 @@ import {
...
   )
 );
 
 function App() {
   return (
-    <>
-      <GlobalStyles />
-      <RouterProvider router={router} />
-    </>
+    //<AppContextProvider>// si lo dejo no me muestra la opción de cargos
```
Comment (ANGIECATHERINEP) [inline src/App.tsx:27]: verifying this part, when adding the appcontext, you can view privileges, that is, if it is working, you must add it

File: src/components/PageTitle/index.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { Stack, Text, useMediaQuery, Icon } from "@inube/design-system";
+import { MdArrowBack } from "react-icons/md";
```
Comment (ANGIECATHERINEP) [inline src/components/PageTitle/index.tsx:2]: you must use the new inubekit library for these components
https://design.inube.cloud

File: src/components/feedback/LogoutModal/styles.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
```
Comment (ANGIECATHERINEP) [inline src/components/feedback/LogoutModal/styles.ts:1]: you must use the new inubekit library for these components

File: src/components/feedback/RenderMessage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import { Stack, SectionMessage } from "@inube/design-system";
```
Comment (ANGIECATHERINEP) [inline src/components/feedback/RenderMessage/index.tsx:1]: you must use the new inubekit library for these components

File: src/components/layout/AppMenu/index.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import { Breadcrumbs, Stack } from "@inube/design-system";
```
Comment (ANGIECATHERINEP) [inline src/components/layout/AppMenu/index.tsx:1]: you must use the new inubekit library for these components

File: src/components/layout/AppMenuGrid/index.tsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import { AppMenuCard } from "@components/cards/AppMenuCard/index";
+import { Grid, useMediaQuery } from "@inube/design-system";
```
Comment (ANGIECATHERINEP) [inline src/components/layout/AppMenuGrid/index.tsx:2]: you must use the new inubekit library for these components

File: src/components/layout/AppPage/index.tsx [hunk 8]
```diff
@@ -24,30 +31,96 @@ const renderLogo = (imgUrl: string) => {
...
-          logoURL={renderLogo(linparLogo)}
-          userName={"Dora Lucia"}
-          client={"Selsa"}
-        />
+        <StyledHeaderContainer>
+          <Header
+            portalId="portal"
+            navigation={nav}
+            logoURL={renderLogo(linparLogo)}
+            userName={"Dora Lucia"}
```
Comment (ANGIECATHERINEP) [inline src/components/layout/AppPage/index.tsx:78]: this data must come from the appContext

File: src/components/navigation/MenuItem/index.tsx [hunk 9]
```diff
@@ -0,0 +1,74 @@
+import { Icon, Text, Stack, inube } from "@inube/design-system";
```
Comment (ANGIECATHERINEP) [inline src/components/navigation/MenuItem/index.tsx:1]: The inubekit library must be used for these components

File: src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx [hunk 10]
```diff
@@ -0,0 +1,144 @@
+import { forwardRef, useEffect, useImperativeHandle, useState } from "react";
+import { GeneralInformationFormUI } from "./interface";
+import { FormikProps, useFormik } from "formik";
+import * as Yup from "yup";
+import { validationMessages } from "@src/validations/validationMessages";
+import { IMessageState } from "@pages/privileges/outlets/users/types/forms.types";
+import { generalMessage } from "../../add-position/config/messages.config";
+import { IHandleUpdateDataSwitchstep } from "../../add-position/types";
+//import { updateItemData } from "@src/mocks/utils/dataMock.service";
```
Comment (ANGIECATHERINEP) [inline src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx:9]: you must delete the comments

File: src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+
+    const formik = useFormik({
+      initialValues,
+      validationSchema,
+      validateOnBlur: false,
+      onSubmit: () => {
+        setLoading(true);
+        setTimeout(() => {
+          handleSubmit && handleSubmit(formik.values);
+          //editDataPosition();
```
Comment (ANGIECATHERINEP) [inline src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx:68]: you must delete the comments

File: src/services/users/invitation.types.ts [hunk 12]
```diff
@@ -0,0 +1,23 @@
+import { IOptionInitialiceEntry } from "@src/pages/privileges/outlets/positions/add-position/types";
```
Comment (ANGIECATHERINEP) [inline src/services/users/invitation.types.ts:1]: you must enter the alias @page

File: .env [hunk 13]
```diff
@@ -0,0 +1,9 @@
+VITE_USERS_URI=http://localhost:3000/privileges/users
+VITE_CLIENTS_URI=http://localhost:3000/Login/11/checking-credentials
+VITE_AUTH0_CLIENT_ID= zXnfpFJ5tS4eyiOTKVfRCKyTVfXxLNBV
+VITE_AUTH_CLIENT_SECRET= jBuivEBCh44ThfSlIEgdoRayL4HeJLJYdpikUcZFMd_sb-DhP5_MucDKLE2vtiCg
+VITE_AUTH0_DOMAIN= dev-rua6a5711i0e17d1.us.auth0.com
+VITE_ICLIENT_API_URL_QUERY= http://localhost:8082/iprivileges-linix-query-data-service/api
+VITE_AUTH0_REDIRECT_URI= https://isetting.vercel.app/
+VITE_ICLIENT_API_URL_QUERY_PROCESS=http://172.106.48.22:8303/iutilities-linix-catalogos-generales-query-process-service/api
+VITE_ICLIENT_API_URL_PERSISTENCE=http://172.106.48.22:8304/iutilities-linix-catalogos-generales-persistence-process-service/api
```
Comment (ANGIECATHERINEP) [inline .env:9]: You should not upload this file to the repository

**Review by ANGIECATHERINEP — CHANGES_REQUESTED** [review]
you need the positions icon
image

and you needed to import the google fonts in the index.html file

File: src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+
+    /*const editDataPosition = async () => {
+      await updateItemData({
+        key: "k_Grupo",
+        nameDB: "linix-positions",
+        identifier: id!,
+        editData: formik.values,
+      });
+    };*/
+
```
[the remark this answers is not in the document]
Comment (SelsaErojas) [reply src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx:59]: comments removed

File: src/App.tsx (hunk 1 above)
Comment (SelsaErojas) [reply src/App.tsx:10]: comments removed, pendding new version Auth

File: src/App.tsx (hunk 2 above)
Comment (SelsaErojas) [reply src/App.tsx:27]: pendding new version Auth

File: src/components/PageTitle/index.tsx (hunk 3 above)
Comment (SelsaErojas) [reply src/components/PageTitle/index.tsx:2]: ok. the change was made

File: src/components/feedback/LogoutModal/styles.ts (hunk 4 above)
Comment (SelsaErojas) [reply src/components/feedback/LogoutModal/styles.ts:1]: ok. the change was made

File: src/components/feedback/RenderMessage/index.tsx (hunk 5 above)
Comment (SelsaErojas) [reply src/components/feedback/RenderMessage/index.tsx:1]: ok. the change was made

File: src/components/layout/AppMenu/index.tsx (hunk 6 above)
Comment (SelsaErojas) [reply src/components/layout/AppMenu/index.tsx:1]: ok. the change was made

File: src/components/layout/AppMenuGrid/index.tsx (hunk 7 above)
Comment (SelsaErojas) [reply src/components/layout/AppMenuGrid/index.tsx:2]: ok. the change was made

File: src/components/layout/AppPage/index.tsx (hunk 8 above)
Comment (SelsaErojas) [reply src/components/layout/AppPage/index.tsx:78]: pendding new version Auth

File: src/components/navigation/MenuItem/index.tsx (hunk 9 above)
Comment (SelsaErojas) [reply src/components/navigation/MenuItem/index.tsx:1]: ok. the change was made

File: src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx (hunk 10 above)
Comment (SelsaErojas) [reply src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx:9]: comments removed

File: src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx (hunk 11 above)
Comment (SelsaErojas) [reply src/pages/privileges/outlets/positions/components/GeneralInformationForm/index.tsx:68]: comments removed

File: src/services/users/invitation.types.ts (hunk 12 above)
Comment (SelsaErojas) [reply src/services/users/invitation.types.ts:1]: done

File: .env (hunk 13 above)
Comment (SelsaErojas) [reply .env:9]: ok. i get it

**Review by ANGIECATHERINEP — APPROVED** [review]

File: src/pages/privileges/outlets/users/types/forms.types.ts [hunk 15]
```diff
@@ -0,0 +1,25 @@
+import { IInvitationsEntry } from "@services/users/invitation.types";
```
Comment (Dlfp) [inline src/pages/privileges/outlets/users/types/forms.types.ts:1]: El subdirectorio /user/types y sus archivos no son necesarios en este proyecto ???

File: src/services/users/invitation.types.ts [hunk 16]
```diff
@@ -0,0 +1,23 @@
+import { IOptionInitialiceEntry } from "@pages/privileges/outlets/positions/add-position/types";
```
Comment (Dlfp) [inline src/services/users/invitation.types.ts:1]: El directorio /services/users y sus archivos no son necesario en este proyecto ....

**Review by ANGIECATHERINEP — APPROVED** [review]

**Review by Dlfp — APPROVED** [review]

**Review by ANGIECATHERINEP — APPROVED** [review]

**Pull request merged**